composer.json

```diff
@@ -3,7 +3,6 @@
   "description": "Sailthru Module for Magento",
   "type":"magento2-module",
   "require": {
-    "php": "~5.6.0|~7.0.0",
```

Comment:
@ladonenko do not remove PHP version restrictions, instead extend the list of those versions:
```
"php": "~5.6.0|~7.0.0|~7.1.3|~7.2.0",
```

Reply:
done